webpack.config.js

```diff
@@ -8,320 +8,327 @@ const webpack = require("webpack");
 let og_image_url = process.env.RIOT_OG_IMAGE_URL;
 if (!og_image_url) og_image_url = 'https://riot.im/app/themes/riot/img/logos/riot-im-logo-black-text.png';
 
-module.exports = (env, argv) => ({
-    entry: {
-        "bundle": "./src/vector/index.js",
-        "indexeddb-worker": "./src/vector/indexeddb-worker.js",
-        "mobileguide": "./src/vector/mobile_guide/index.js",
+module.exports = (env, argv) => {
```

Comment:
this diff is ugly, but what has happened is the `({...})` bit has turned into a real function so the `if` statement below can be included. The entire config object has tabbed 1 to the right to account for the new `return {...}`.